components/coding/studentPortal/ProfileDetailCard.tsx

```diff
@@ -16,12 +19,13 @@ type ProfileDetailCard = {
 function ProfileDetailCard({
   avatar,
   name,
-  link,
   joinDate,
   badges,
   currentBadge,
   nextGoal,
 }: ProfileDetailCard) {
+  const { totalBadgeCount } = useSelector(profileSelector);
```

Comment:
We should pass this as a prop like the other props here like currentBadge, name, etc...

Let's not use the selector here.

Reply:
fed it in as a prop. thanks!